common/src/mock.rs

```diff
@@ -1090,8 +1090,10 @@ macro_rules! mock_pswap_distribution_config {
         $buy_back_handler:ty,
         $ensure_dex_manager:ty
     ) => {
+        use sp_runtime::Permill;
         frame_support::parameter_types! {
             pub GetIncentiveAssetId: AssetId = common::PSWAP.into();
+            pub GetBuyBackFractions: Vec<(AssetId, Permill)> = vec![(common::KUSD.into(), Permill::from_rational(39u32, 100u32)), (common::TBCD.into(), Permill::from_rational(1u32, 100u32))];
```

Comment:
Maybe won't be so clear, but not need to calculate total_fraction and simplify code if use:
`Permill::from_rational(975u32, 1000u32), Permill::from_rational(25u32, 1000u32)`